frontend/src/components/slc-configurate-function/slc-configurate-function.tsx

```diff
@@ -0,0 +1,73 @@
+import { FC } from 'react';
+import { useAppDispatch, useAppForm } from 'hooks/hooks';
+import { SLCFunctionCreateRequestDto } from 'common/types/types';
+import { DEFAULT_PAYLOAD } from './common/constants';
+import { slcFunctionCreate as CreateSLCFunctionValidationSchema } from 'validation-schemas/validation-schemas';
+import { SLCFunctionConfigurate as SLCFunctionConfigurateActions } from 'store/actions';
+import {
+  AppRoute,
+  ButtonStyle,
+  ButtonType,
+  InputType,
+} from 'common/enums/enums';
+import { Button, Input } from 'components/common/common';
+import { getNameOf } from 'helpers/helpers';
+
+import styles from './styles.module.scss';
+
+const SLCConfigurateFunction: FC = () => {
+  const dispatch = useAppDispatch();
+
+  const { control, errors, handleSubmit } =
+    useAppForm<SLCFunctionCreateRequestDto>({
```

Comment:
Should i use here SLCFunctionCreateFormDto?
```suggestion
    useAppForm<SLCFunctionCreateFormDto>({
```

Reply:
if they are the same for both backend and frontend you can use one `SLCFunctionCreateRequestDto `